public/js/dbkjs/layers.js

```diff
@@ -233,6 +235,9 @@ dbkjs.layers = {
           var myLayer;
           var onlySearch = wms_v.options && wms_v.options.wfsSearch && wms_v.options.wfsSearch.onlySearch;
           var index = wms_v.index || 0;
+          if (dbkjs.options.useAPIforWMS) {
```

Comment:
There actually already was a  column `organisation.wms.proxy` for specifying it per-wms instead of globally.